addons/pos_self_order/static/tests/utils/tour_utils.js

```diff
@@ -191,6 +238,34 @@ export const PosSelf = {
             ];
         },
         selectAttributes: (attributes) => attributes.map((attribute) => attributeHelper(attribute)),
+        clickKioskTrash: (name) => {
+            return [
+                {
+                    content: `Click on trash icon`,
+                    trigger: `.o_kiosk_item_card:has(.o_kiosk_product_name:contains("${name}")) .btn .fa.fa-trash`,
+                }
+            ];
+        },
+        pressNumpad: (keys) => {
+            const numberChars = "0 1 2 3 4 5 6 7 8 9".split(" ");
+            function generateStep(key) {
+                let trigger;
+                if (numberChars.includes(key)) {
+                    trigger = `.numpad .touch-key:contains("${key}")`;
+                }
+                return {
+                    content: `'${key}' pressed in numpad`,
+                    trigger,
+                };
+            }
+            return keys.split(" ").map(generateStep);
+        },
```

Comment:
You can use the function from `point_of_sale/static/tests/tours/helpers/NumpadTourMethods.js` even though the kiosk doesn't yet use the generic numpad. Just refactor it such that it takes the `trigger` as an option.